deploy/your-repo-name/values/production.yaml

```diff
@@ -44,26 +44,24 @@ securityContext:
     type: RuntimeDefault
 
 env:
-  # These will be overridden by deploy.sh with values from GitHub vars/secrets
-  SERVICE_NAME: ""
-  SERVICE_PHASE: ""
-  DEPARTMENT_NAME: ""
-  DEPARTMENT_URL: ""
-  CONTACT_EMAIL: ""
+  SERVICE_NAME: "Express TypeScript Template"
```

Comment:
We should have consistent env values across the repo. It will get confusing for someone to look a this otherwise. Can we have the following:
```
SERVICE_NAME: "Your Service Name"
SERVICE_PHASE: "Alpha"
DEPARTMENT_NAME: "LAA Digital"
DEPARTMENT_URL: "https://www.gov.uk/government/organisations/legal-aid-agency"
CONTACT_EMAIL: "your-team@justice.gov.uk"
```